templates/project/.github/PULL_REQUEST_TEMPLATE.md.twig

```diff
@@ -3,13 +3,15 @@
 
 <!-- Describe your Pull Request content here -->
 
+{% if project.stableBranch is not null %}
 <!--
     Show us you choose the right branch.
     Different branches are used for different things :
     - {{ project.stableBranch.name }} is for everything backwards compatible, like patches, features and deprecation notices
     - {{ project.unstableBranch.name }} is for deprecation removals and other changes that cannot be done without a BC-break
     More details here: https://github.com/sonata-project/{{ project.repository.name }}/blob/{{ project.stableBranch.name }}/CONTRIBUTING.md#base-branch
 -->
+{% endif %}
 I am targeting this branch, because {reason}.
```

Comment:
Can we move this line inside the `stableBranch` block?

Reply:
Done